src/status_im/ui/screens/navigation.cljs

```diff
@@ -49,9 +49,13 @@
 
 (fx/defn navigate-reset
   [{:keys [db]} {:keys [index actions] :as config}]
-  (let [view-id (:routeName (get actions index))]
-    {:db              (assoc db :view-id view-id
-                             :navigation-stack (list view-id))
+  (let [stack (into '() (map :routeName actions))
```

Comment:
why list?

Reply:
it is explained in the note below, navigate back uses `pop` which only works on list not lazyseq